index.js

```diff
@@ -0,0 +1,101 @@
+
+function defineSession(name) {
+  var EditSession = require("ace/edit_session").EditSession;
+  var html = new EditSession(`<!DOCTYPE html>
+  <html>
+  <head>
+  </head>
+
+  <body>
+  </body>
+
+  </html>`, 1);
+  var css = new EditSession(["some", "css", "code here"]);
```

Comment:
what is the array inside for? doesn't seem like it's used anywhere with those values.